.github/actions/bootstrap/action.yml

```diff
@@ -16,6 +16,25 @@ runs:
         path: ~/.mint
         key: ${{ runner.os }}-mint-v1-${{ hashFiles('Mintfile') }}
 
+    - name: Brew cache params
+      shell: bash
+      id: brew-cache
+      run: |
+        deps=$(sed "s/brew '\(.*\)'.*/\1/" Brewfile | grep -v 'gh')
+        brew_prefix="$(brew --prefix)/Cellar"
+        paths=$(echo "$deps" | sed "s#^#$brew_prefix/#g")
+        cache_key=$(echo $deps | tr -d '\n' | sed 's/ /-/g')
+        echo "cache-key=$cache_key" >> $GITHUB_OUTPUT
+        echo "paths<<EOF" >> $GITHUB_OUTPUT
```

Comment:
what is $GITHUB_OUTPUT and does the `EOF` string have special meaning to it? I'm reading this as if it's writing to a file and it would just put the literal characters there?

Reply:
GITHUB_OUTPUT allows you to write outputs (key/value pairs) from one step to be used in another step, and the EOF is a technique for doing this for multiline text (similar to [Here Documents](https://tldp.org/LDP/abs/html/here-docs.html))

Reply:
https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs